adapters/consumable/consumable.go

```diff
@@ -77,6 +77,10 @@ type decision struct {
 	ImpressionUrl *string    `json:"impressionUrl,omitempty"`
 	Width         uint64     `json:"width,omitempty"`  // Consumable extension, not defined by Adzerk
 	Height        uint64     `json:"height,omitempty"` // Consumable extension, not defined by Adzerk
+	Adomain       []string   `json:"adomain,omitempty"`
+	Cats          []string   `json:"cats,omitempty"`
+	NetworkId     int        `json:"networkId,omitempty"`
```

Comment:
I don't see the `NetworkId` and `MediaType` value being used anywhere. Are they still needed?

Reply:
Missed out on removing those. Should be updated now to reflect changes.

Reply:
Perfect. Thanks @jpiros!